src/components/ExperienceTable/ExperienceTableRow.tsx

```diff
@@ -23,85 +32,143 @@ const Title = ({ nested, row }: { nested?: boolean; row: ExperienceTableRowData
   const { i18n } = useTranslation();
 
   return nested ? (
-    <span className="px-5 text-black text-body-sm sm:pt-3 font-bold sm:font-normal sm:text-body-md">
+    <p className="pl-5 pr-7 pt-2 text-body-sm font-bold sm:font-normal sm:text-body-md sm:py-2">
       {row.nimi[i18n.language]}
-    </span>
+    </p>
   ) : (
-    <span className="px-5 text-heading-4 sm:text-heading-3">{row.nimi[i18n.language]}</span>
+    <p className="pl-5 pr-7 pt-2 text-heading-4 sm:text-heading-3 sm:pt-1 sm:pb-[3px]">{row.nimi[i18n.language]}</p>
```

Comment:
On mobile, the line height is `27px`. It might be wanted to give the title little more room as the informations is so compact.

Reply:
Should be as in Figma.